string id
string admin_code
int32 road_rank
int32 road_type
string road_no
int32 link_type
int32 lane_no
string r_link_id
string l_link_id
string from_node_id
string to_node_id
string section_id
float64 length
string its_link_id
string maker
string update_date
string version
string remark
string hist_type
string hist_remark